ktx/lifecycle-livedata-ktx/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ## Unreleased
 
+### Changes
+
+- Move `EventQueue` to a separate module and rename it to `ViewModelEvents`. See appropriate [documentation](../../viewmodelevents/README.md)
```

Comment:
By the way, have we deleted `EventQueue` and `Event` from this module?
```suggestion
- **Breaking change!** `EventQueue` moved to a separate module and renamed to `ViewModelEvents`. See appropriate [documentation](../../viewmodelevents/README.md)
```

Reply:
Yes, we did